mender-auth/cli/cli.cpp

```diff
@@ -72,27 +72,33 @@ static ExpectedActionPtr ParseUpdateArguments(
 		return expected::unexpected(conf::MakeError(conf::InvalidOptionsError, "Need an action"));
 	}
 
-	if (start[0] == "daemon") {
-		string passphrase = "";
-		conf::CmdlineOptionsIterator opts_iter(start + 1, end, {"--passphrase-file"}, {});
-		auto ex_opt_val = opts_iter.Next();
-
-		while (ex_opt_val
-			   && ((ex_opt_val.value().option != "") || (ex_opt_val.value().value != ""))) {
-			auto opt_val = ex_opt_val.value();
-			if ((opt_val.option == "--passphrase-file")) {
-				auto ex_passphrase = GetPassphraseFromFile(opt_val.value);
-				if (!ex_passphrase) {
-					return expected::unexpected(ex_passphrase.error());
-				}
-				passphrase = ex_passphrase.value();
+	string passphrase = "";
+	bool forcebootstrap = false;
+	conf::CmdlineOptionsIterator opts_iter(
+		start + 1, end, {"--passphrase-file"}, {"--forcebootstrap", "-F"});
```

Comment:
It's a bit confusing that later arguments are checked before former arguments, and I think this can very easily break if we add any commands later. I suggest checking the command value first (`daemon` or `bootstrap`), do the flag parsing inside the if-block, and then check them again afterwards to pick one of them.